packages/tables/resources/views/columns/select-column.blade.php

```diff
@@ -96,6 +96,7 @@
                 isLoading = false
             "
             :attributes="\Filament\Support\prepare_inherited_attributes($getExtraInputAttributeBag())"
+            wire:loading.attr="disabled"
```

Comment:
Does this work reliably with line 78? I believe that was the main challenge.

Reply:
Yeah please check if you can :)

Reply:
Thanks for reminding me. I completely forgot 😅. I will check as soon as I can!

Reply:
Oops I accidentally deleted the repository. However, i have some free time now. I will create a new repo and investigate. Apologies for the delay :sweat_smile: